mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -65,6 +74,22 @@
 
   {% include "cashbook/includes/batch_buttons.html" %}
 
+  <dialog id="print-dialog" class="Dialog">
+    <div class="Dialog-inner">
+      {% blocktrans %}
+      <h3>Before you print</h3>
+      <p>Have you tried changing the size of the window so you can see the payments and NOMIS at the same time?</p>
+      <p>You could also try using the ALT + TAB keys to switch between the windows.</p>
+      {% endblocktrans %}
+      <button class="button u-nojs-hidden js-Print">{% trans "Print now" %}</button>
+
+      <p>
+        <input type="checkbox" name="remove-print-prompt" id="remove-print-prompt" value="1">
+        <label for="remove-print-prompt">{% trans "Don't show this message again" %}</label>
+      </p>
+    </div>
+  </dialog>
```

Comment:
It might be nice to add these bits of JS-only print UI with javascript, rather than in HTML.